meta/templates/meta/meta.html

```diff
@@ -10,7 +10,8 @@
         {% if meta.title %}{% title_prop meta.title %}{% endif %}
     {% endif %}
     {% if meta.use_og %}
-        {% if meta.title %}{% og_prop 'title' meta.title %}{% endif %}
+        {% if meta.og_title %}{% og_prop 'title' meta.og_title %}
+        {% elif meta.title %}{% og_prop 'title' meta.title %}{% endif %}
```

Comment:
I wonder if we could handle the fallback at the python level.
Changing the template it's always a bit nasty because it requires people who overrode this, to sync with upstream changes.
In this case they will need to update to use the new properties, but if we keep the fallback at the python level, if we change something in how the fallback is handled, it will make easier for other people to update

Reply:
Agree, we can't, unfortunately